src/main/java/com/mixtape/mixtapeapi/playlist/PlaylistRepository.java

```diff
@@ -0,0 +1,6 @@
+package com.mixtape.mixtapeapi.playlist;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
```

Comment:
```suggestion
@Repository
```
These need to be annotated with `@Repository` so that they get autowired